src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -1290,6 +1294,57 @@ def __print_kvset_json_diff(old_obj=None, new_obj=None, yes=False):
     return True
 
 
+def __tags_equals(tag1, tag2):
+
+    if not tag1 and not tag2:
+        return True
+
+    if not (isinstance(tag1, dict) and isinstance(tag2, dict)):
+        return False
+
+    if len(tag1) != len(tag2):
+        return False
+
+    try:
+        for tag_key in tag1.keys():
+            if tag1[tag_key] != tag2[tag_key]:
+                return False
+
+        return True
+
+    except KeyError:
+        return False
+
+
+# Compare the key-value fields to determine whether two key-values are the same.
+# We use the specified label and content_type values for key-values being imported.
+# Feature flag content-type is constant so we use this for feature flag comparison instead of the input content_type argument.
+def __kv_equals(source_kv, dest_kv, input_label, input_content_type):
```

Comment:
Just to confirm - there's no SDK overload for checking if two configuration settings are equal?